Webpack docs

jsprw opened the pull request:
Added webpack section to deployment documentation.

B4nan commented:
Looking good, I just don't like this sentence very much `If you want to deploy the least amount of files possible, you should use Webpack`. Personally I see this webpack solution viable only for angular universal or electron apps where the result is actually ran in browser based environment. Generally I would not suggest to use webpack (or any other build tool made for frontend) when its needed. 

Also please use `MikroORM` instead of `Mikro-ORM`. `mikro-orm` is the package name, `MikroORM` is the library name. I don't want to promote that mixture of both :]

jsprw commented:
> Personally I see this webpack solution viable only for angular universal or electron apps where the result is actually ran in browser based environment.

[Angular universal](https://angular.io/guide/universal) is not run in browser based environments. It's run on a node server and produces data to be sent to the browser environment. 

> Generally I would not suggest to use webpack (or any other build tool made for frontend) when its not needed.

In my opinion, it works well as you only need to install node on the server and no other node modules. Furthermore the plugins are nice to have. Anyways, I have removed the sentence.

> Also please use MikroORM instead of Mikro-ORM. mikro-orm is the package name, MikroORM is the library name. I don't want to promote that mixture of both :]

Fixed.

B4nan commented:
> Angular universal is not run in browser based environments. It's run on a node server and produces data to be sent to the browser environment.

Yeah I know how that works, I am using that myself for a long time already. But my point is that it is forcing you to use webpack, or not? 

Never mind, LGTM, thanks!
